docs/release/Betaflight-3.0.x-Release-Notes.md

```diff
@@ -1,5 +1,5 @@
 
-# BetaFlight 3.0 Release Notes
+# BetaFlight 3.0.x Release Notes
```

Comment:
Imho we should not include patch version (only major.minor)

Reply:
Okay, fixed :+1: